<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fihm</class>
 <widget class="QMainWindow" name="fihm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>fihm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>1920</width>
     <height>1080</height>
    </size>
   </property>
   <widget class="QWebView" name="webViewStarter">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>811</height>
     </rect>
    </property>
    <property name="url">
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
   <widget class="QWebView" name="webViewFlash">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>811</y>
      <width>1681</width>
      <height>264</height>
     </rect>
    </property>
    <property name="url">
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
   <widget class="QLabel" name="lbHeure">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>820</y>
      <width>51</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbPoints">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>820</y>
      <width>16</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbTemp">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>870</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>29.5</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbTemp_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>870</y>
      <width>51</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string notr="true">Â°C</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbMinute">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>820</y>
      <width>51</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
